reoptjl/models.py

```diff
@@ -6193,6 +6371,55 @@ def clean(self):
         if self.addressable_load_fraction == None:
             self.addressable_load_fraction = list([1.0]) # should not convert to timeseries, in case it is to be used with monthly_mmbtu or annual_mmbtu
 
+class ProcessHeatLoadInputs(BaseModel, models.Model):
+    # DHW
```

Comment:
Is "#DHW" here intentional?

Reply:
It is not!  Updated it here - https://github.com/NREL/REopt_API/commit/d7d6652c4dd3dcc0970c7305c035673fd374f744